### DISCUSSION TITLE
Frame Z index

### DISCUSSION THREAD

**Discussion by Vespertellino** [keiwando/toonsquid#529] [Ideas]
There's this incredible feature in Aseprite called Z index, here's a brief demonstration on how it works: https://youtu.be/yT4q9QmdYfY?t=183
Basically it lets a frame or group of frames appear as if they were on a different layer (higher or lower)
For example when I need something to be on top of an object for most of the animation, but then for a few frames I'd like it to be behind it - instead of creating a new layer below and increаsing clutter, I could just change Z indeх for several frames.
I think that would be a great addition, especially when animating complex scenes, as layer count can go very high.

**Comment by Cat-Butler** [comment]
See also: https://github.com/keiwando/toonsquid/discussions/71 (+10) and mentioned in https://github.com/keiwando/toonsquid/discussions/140

**Comment by DWrightAnimation** [comment]
This feature is a must for those who want to save as many layers as possible, but in the meantime, the closest thing to achieving this effect is sandwiching the object being orbited between 2 layers of the same thing, then, when it's supposed to go behind the center of the orbit, have the opacity of the top layer turned all the way down in the frames where it's behind, which can be done by keyframing the opacity, and having the easing set to hold.
